requirements.txt

```diff
@@ -0,0 +1,33 @@
+numpy==1.23.2
```

Comment:
This isn't the worst thing in the world, but I think its cleaner if you only include the things you would have `pip installed`'d if you weren't using this `requirements.txt` file. 

In your case, I think thatd be: 

numpy
matplotlib (even though you haven't used it yet...)
pandas

(plus their version numbers)